src/ClassyResource/createMethod.js

```diff
@@ -87,12 +87,19 @@ export default function createMethod(spec) {
       }
 
       if (response === 'member') {
+
+        let baseUrl = this._classy.baseUrl;
+        //  Gateway url to support okta based member token
+        if (_.get(data, 'token.is_okta_token', false)) {
```

Comment:
Can you add a space above the comment here? We should have spaces before and after every multi-line statement unless at the beginning or end of a function, etc.

Reply:
Done